enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -31,4 +31,8 @@ export const SdkContentWrapper = styled.div<{ font: string }>`
 
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
+  svg {
```

Comment:
I'm note sure I understand why we need this, I haven't found equivalent code in the non-sdk code and the few svgs I inspected had `display: block`;
Is it a fix for some 3rd party library for which we don't include the css in the sdk?

Reply:
> display: inline for svgs is default browser behavior

Ah that's what I missed 😅, makes sense then